examples/research_projects/wav2vec2/alignment.py

```diff
@@ -0,0 +1,209 @@
+import argparse
+import torchaudio
+import transformers
+import os
+import torch 
+from dataclasses import dataclass
+from tqdm import tqdm 
```

Comment:
Could we add a comment here that the code was "heavily inspired" by https://pytorch.org/audio/stable/tutorials/forced_alignment_tutorial.html ? 

Would be nice to give the torchaudio team some credit here in case you used it ;-)